obspy/taup/tests/test_tau.py

```diff
@@ -16,6 +16,11 @@
 import numpy as np
 
 from obspy.taup import TauPyModel
+# We need HAS_GEOGRAPHICLIB, but we need to be able to set
+# in all three places... there must be a better way!
+import obspy.geodetics.base as geobase
```

Comment:
This import is really obfuscating.. you only ever use `HAS_GEOGRAPHICLIB` from `obspy.geodetics.base`. So why not make it simply `from obspy.geodetics.base import HAS_GEOGRAPHICLIB` and avoid inventing new submodule aliases?